examples/hybrid/sphere/single_column_radiative_equilibrium.jl

```diff
@@ -0,0 +1,98 @@
+using PrettyTables
```

Comment:
This kind of goes against the grain of #339, but I don't want to slow things down..

Reply:
After I'm done with radiation, I'd like to do a PR that organizes all our `examples` code into modules and defines the "default driver setups" we discussed earlier. That should fix this issue.